### PR TITLE
NO-JIRA: minor logging update when data gathering is disabled

### PR DESCRIPTION
**Pull request by tremes**
This PR implements a new data enhancement to...

## Categories

- [ ] Bugfix
- [ ] Data Enhancement
- [ ] Feature
- [ ] Backporting
- [X] Others (CI, Infrastructure, Documentation)

## Sample Archive

- `path/to/sample_data.json`

## Documentation

- `path/to/documentation.md`

## Unit Tests

- `path/to/file_test.go`

## Privacy

Yes. There are no sensitive data in the newly collected information.

## Breaking Changes

Yes/No

## References

https://issues.redhat.com/browse/???
https://access.redhat.com/solutions/???

### CODE REVIEW

File: pkg/insights/insightsuploader/insightsuploader.go [hunk 1]
```diff
@@ -69,83 +69,70 @@ func New(summarizer Summarizer,
...
-	lastReported := c.reporter.LastReportedTime()
-	if !lastReported.IsZero() {
-		next := lastReported.Add(interval)
-		if now := time.Now(); next.After(now) {
-			c.initialDelay = wait.Jitter(next.Sub(now), 1.2)
-		}
-	}
-	klog.Infof("Reporting status periodically to %s every %s, starting in %s", cfg.DataReporting.UploadEndpoint, interval, c.initialDelay.Truncate(time.Second))
+	// set the initial upload delay as initial delay + 2 minutes
+	ud := 90 * time.Second
```
Comment (ncaak) [inline pkg/insights/insightsuploader/insightsuploader.go:83]: Isn't two minutes (per the comment above) 120 seconds?

**Review by ncaak — APPROVED** [review]
/lgtm

File: pkg/insights/insightsuploader/insightsuploader.go (hunk 1 above)
Comment (tremes) [reply pkg/insights/insightsuploader/insightsuploader.go:83]: :D correct. Thanks. I am going to change the comment to 1m30s :)

**Pull request merged**